cmd/beaker/session.go

```diff
@@ -244,36 +258,50 @@ Do not write sensitive information outside of the home directory.
 		}
 		shouldCancel = false
 
-		if saveImage {
-			var job *api.Job
-			started := func(ctx context.Context) (bool, error) {
-				var err error
-				job, err = beaker.Job(session.ID).Get(ctx)
-				if err != nil {
-					return false, err
-				}
-				return job.Status.Finalized != nil, nil
-			}
-			if err := await(ctx, "Waiting for image capture to complete", started, 0); err != nil {
-				return fmt.Errorf("waiting for image capture to complete: %w", err)
-			}
-			if job.Status.Failed != nil {
-				return fmt.Errorf("session failed: %s", job.Status.Message)
-			}
-			images, err := beaker.Job(job.ID).GetImages(ctx)
+		if !saveImage {
+			return nil
+		}
+		var job *api.Job
+		started := func(ctx context.Context) (bool, error) {
+			var err error
+			job, err = beaker.Job(session.ID).Get(ctx)
 			if err != nil {
-				return err
-			}
-			if len(images) == 0 {
-				return fmt.Errorf("job has no result images")
+				return false, err
 			}
+			return job.Status.Finalized != nil, nil
+		}
+		if err := await(ctx, "Waiting for image capture to complete", started, 0); err != nil {
+			return fmt.Errorf("waiting for image capture to complete: %w", err)
+		}
+		if job.Status.Failed != nil {
+			return fmt.Errorf("session failed: %s", job.Status.Message)
+		}
+		images, err := beaker.Job(job.ID).GetImages(ctx)
+		if err != nil {
+			return err
+		}
+		if len(images) == 0 {
+			return fmt.Errorf("job has no result images")
+		}
+		if !quiet {
+			fmt.Printf("Image saved to %s: %s/im/%s\n",
+				color.BlueString(images[0].ID),
+				beaker.Address(),
+				images[0].ID)
+		}
+		if !updateDefaultImage {
 			if !quiet {
-				fmt.Printf(`Image saved to %[1]s: %[2]s/im/%[3]s
-Resume this session with: beaker session create --image beaker://%[3]s
-`, color.BlueString(images[0].ID), beaker.Address(), images[0].ID)
+				fmt.Printf(`Default image not updated.
+Resume this session with: beaker session create --image beaker://%s
+`, images[0].ID)
 			}
+			return nil
 		}
-
+		beakerConfig.DefaultImage = "beaker://" + images[0].ID
+		if err := config.WriteConfig(beakerConfig, config.GetFilePath()); err != nil {
+			return fmt.Errorf("setting default image: %w", err)
+		}
+		fmt.Println("Default image updated. Resume this session with: beaker session create")
```

Comment:
```suggestion
		fmt.Printf("Default image updated in your config file (%s). Resume this session with: beaker session create\n", config.GetFilePath())
```

Is "resume" the right verb?